shelley/chain-and-ledger/formal-spec/Properties.md

```diff
@@ -256,13 +292,19 @@ the epoch nonce is what you get from combining the blocks leading up to it
 The overlay schedule is obeyed: no blocks are produced during the silent blocks,
 and only core nodes makes blocks on the overlay slots.
 
+**Property** Full decentralization is reached eventually
+
 # Rewards Properties
 
 **Property**
 At the start of each epoch, the reward update is set to NOTHING.
 Moreover, the reward update will change exactly once during the epoch,
 to a non-NOTHING value.
 
+**Property**
+Total rewards distributed at the epoch boundary is the amount in the reward pot
+minus the share calculated for stake pools that did not meet their pledge.
```

Comment:
even if every pool meets its pledge, some of the reward pot can end up in the treasury.